### ISSUE TITLE
Returning more (or less) than 1 row per `summarise()` group was deprecated in dplyr 1.1.0.

### ISSUE THREAD

**Issue by remip48** [gavinsimpson/gratia#225]
Hello,

With the new update of Dplyr, summarise function has changed and can return only one row per group (otherwise returning an error message at the moment (see below) and an error in the future). This is due to the new function reframe, which return here more than one row per group. So gratia::draw(my_model) return in error with mgcv::gam models now.

See new warning:
Warning message:
Returning more (or less) than 1 row per `summarise()` group was deprecated in dplyr 1.1.0.
ℹ Please use `reframe()` instead.
ℹ When switching from `summarise()` to `reframe()`, remember that `reframe()` always returns an ungrouped data frame and
  adjust accordingly.
ℹ The deprecated feature was likely used in the gratia package.
  Please report the issue at <https://github.com/gavinsimpson/gratia/issues>.
This warning is displayed once every 8 hours.
Call `lifecycle::last_lifecycle_warnings()` to see where this warning was generated. 

Best regards

**Comment by gavinsimpson** [comment]
What version of gratia was this issue observed with? I fixed these prior to the *dplyr* release, so if this is with the current CRAN version or the development version then that would be useful to know.

**Comment by remip48** [comment]
Indeed, thank you for your answer and sorry to have raised an issue already solved!

Best
